data.sql

```diff
@@ -4,3 +4,12 @@ VALUES('Agumon', '2020-02-03', 0, true, 10.23),
     ('Gabumon', '2018-11-15', 2, true, 8),
     ('Pikachu', '2021-01-07', 1, false, 15.4),
     ('Devimon', '2017-05-12', 5, true, 11);
+
+// insert more data into our TABLE
+INTO animals(NAME, DATE_OF_BIRTH, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) 
+VALUES('Charmander', '2020-02-08', 0, true, 11),
+    ('Plantmon', '2021-11-15', 2, true, -5.7),
+    ('Angemon', '2005-06-12', 3, false, -45),
+    ('Boarmon', '2005-06-07', 7, true, 20.4),
+    ('Blossom', '1998-10-13', 3, true, 17),
+    ('Ditto', '2022-05-14', 4, true, 22);
```

Comment:
- Good job adding more animals to your table, but you have not correctly inputted the right data, the animal `Charmander` should have a weight of `-11`. Please fix this 👍 👍 
- Also, according to the project requirements you are to have an animal named `Squirtle` in your database that's currently missing in your data. please fix this 👍 👍